superset-frontend/package.json

```diff
@@ -342,6 +342,7 @@
   "peerDependencies": {
     "ace-builds": "^1.41.0",
     "core-js": "^3.38.1",
+    "handlebars": "^4.7.8",
```

Comment:
@mistercrunch Given that `handlebars` is being used by Superset and not only by the plugin anymore, I think we should declare it as a dependency of Superset and make it a peer dependency in the plugin. The current form is problematic as the plugin might not be installed.

Reply:
@mistercrunch Please ignore this comment. I saw that you did exactly what I asked but using the core package which is better.

Reply:
yes if it's needed in src/ (main app), it should be declared as a peerDep